### ISSUE TITLE
Display link hints in expected case

### ISSUE THREAD

**Issue by FPFanatic** [ueokande/vim-vixen#632]
### I'm opening this issue because:

- [ ] I'll report a bug
- [x] I'll propose a new feature

### Description

The current system for links could be more intuitive. Whereas other keybindings are case-sensitive, link hints are not, leading to confusion.

#### Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Press f to start following links
2. Spot a hint, e.g., link A
3. Type capital A
4. Link is not followed
5. Press f again to follow links again
6. Spot link A again
7. Type lowercase a
8. Link followed

#### System configuration

- Operating system: Ubuntu 18.04.2
- Firefox version: 68.0.1
- Vim-Vixen version: 0.23

**Comment by mvhulten** [comment]
I would agree this should be case sensitive.  In that case, I would suggest to choose lower case (for both link labels and keys typed) because it is quicker to type.  You could include digits as well, but not upper case letters.

**Comment by AlexeySachkov** [comment]
> You could include digits as well

@mvhulten, list of symbols used for link hints is actually configurable, see [docs](https://github.com/ueokande/vim-vixen#hintchars) and [default value](https://github.com/ueokande/vim-vixen/blob/master/src/shared/SettingData.ts#L409).
